app/src/main/java/edu/uc/mille5bm/smartnutrition/dto/Food.kt

```diff
@@ -1,9 +1,10 @@
 /** Holds information about Food ..*/
 package edu.uc.mille5bm.smartnutrition.dto
 
+import com.google.gson.annotations.SerializedName
 import java.util.*
 
-class Food(var id: String, var name: String, var nutritionPer100g: String, var tags: String){
+data class Food(@SerializedName("id")var id: String, @SerializedName("name") var name: String, @SerializedName("nutrition-per-100ml") var nutritionPer100g: String, @SerializedName("tags") var tags: String){
```

Comment:
data modifier is good.

@SerializedName:  Is this matching to a JSON feed?  If not, what is the purpose of these annotations?